### DISCUSSION TITLE
Cutlass support compatibility with nv_bfloat

### DISCUSSION THREAD

**Discussion by YoniSp** [NVIDIA/cutlass#547] [Q&A]
Hi I am trying to create matrix and multiply  by using nv_bfloat without success.
There is an option to use nv_bfloat or only using cutlass::bfloat16_t is valid?

**Comment by hwu36** [comment]
`cutlass::bfloat16_t` can be used in both host and device.  It is also compatible with `nv_bfloat`. So, use `cutlass::bfloat16_t` whenever possible.  Use `reinterprect_cast<>` whenever you need to switch.
